_episodes/05-join.md

```diff
@@ -492,8 +492,8 @@ INFO: Query finished. [astroquery.utils.tap.core]
 {: .output}
 
 ~~~
-results = job.get_results()
-results
+cone_results = cone_job.get_results()
+cone_results
```

Comment:
Do you think this should have a different name since we use `cone_results` in episode 2?